src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,70 @@ redirects:
+
+## **Detect** problems in your system [#detect]
+
+Use the detect features to define the conditions in your system you want to be notified about, as well as to configure how anomalies are detected.
+
+* **[Alert conditions (policies)](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/)**: Create new policies. Also, lists the policies where your conditions are created and organized. 
+* **[Anomaly detection](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/)**: Manage your anomaly detection configurations. An anomaly is a behavioral trend that doesn't match the historical data for your system. You can add or edit the configurations here.
+
+## **Correlate** the data coming into your system [#correlate]
+
+Use the correlate features to define your data sources and to review and configure the logic that groups your incidents.
+
+* **[Sources](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources)**: Manage the data input sources you've chosen to analyze and be notified about. You can add new sources or configure existing ones. Sources include New Relic sources, such as alerts conditions and detected anomalies, as well as third-party sources like AWS Cloudwatch Alarms and PagerDuty.
+* **[Decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/)** Review your correlated incidents. You can also review, edit, and add decisions.
+
+## **Enrich and notify** your notifications with useful metadata [#enrich]
+
+Use the enrich and notify features to add additional metadata to your notifications, schedule when you don't want to be notified, and configure where your notifications get sent.
+
+* **[Muting rules](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications/)**: Temporarily stop your alerts notifications.
+* **[Workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/)** Add more data to your alerts notifications.
+* **[Destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#2-configure-destinations)**: Configure where your applied intelligence notifications get sent.
+* **[Pathways](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#3-configure-pathways)**: Control where your correlated issues get sent.
```

Comment:
How are they sent? How is this different than Destinations?

Reply:
Maybe you can clue me in here? Those are great questions!